openstack/keystone/templates/etc/_secrets.conf.tpl

```diff
@@ -20,14 +20,14 @@ connection = {{ include "db_url_mysql" . }}
 [cache]
 memcache_sasl_enabled = True
 memcache_username = {{ .Values.memcached.auth.username }}
-memcache_password = {{ .Values.memcached.auth.password }}
+memcache_password = {{ .Values.memcached.auth.password | include "resolve_secret" }}
```

Comment:
Technically, we would also need to escape here a potential `$` sign, but since it wasn't before, it should be fine.